.werft/vm/manifests.ts

```diff
@@ -41,6 +41,14 @@ spec:
       labels:
         harvesterhci.io/vmName: ${vmName}
     spec:
+      readinessProbe:
+        tcpSocket:
+          port: 22
+        initialDelaySeconds: 120
```

Comment:
120 seconds waiting after the VM has started until the readiness probe starts probing seems like a lot. Why do we want an initial delay at all?

Reply:
From the tests I've made, not a single VM got ready in less than 3 minutes, and from their [docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes)
![image](https://user-images.githubusercontent.com/24193764/148556787-9cdcfb6a-2126-45b9-b823-d308acc33002.png)

FailureThresholds are the amount of times that we try to open the TCP Socket before setting the pod status to Failed. There is no point trying too early if we know that it just won't be ready

Reply:
ok, then let's leave it as is.